<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="5" failed="1" ignored="0" total="16" passed="10">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="843923" started-at="2018-10-16T04:37:40Z" finished-at="2018-10-16T04:51:44Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="843923" started-at="2018-10-16T04:37:40Z" finished-at="2018-10-16T04:51:44Z">
      <class name="CardholderMavenProject.Cardholder">
        <test-method status="SKIP" signature="Registration()[pri:1, instance:CardholderMavenProject.Cardholder@5bcab519]" name="Registration" duration-ms="9" started-at="2018-10-16T04:37:40Z" finished-at="2018-10-16T04:37:40Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping registrationTest case. ]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping registrationTest case. 
at CardholderMavenProject.Cardholder.Registration(Cardholder.java:137)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Registration -->
        <test-method status="PASS" signature="Login()[pri:2, instance:CardholderMavenProject.Cardholder@5bcab519]" name="Login" duration-ms="29759" started-at="2018-10-16T04:37:40Z" finished-at="2018-10-16T04:38:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method status="SKIP" signature="Account()[pri:3, instance:CardholderMavenProject.Cardholder@5bcab519]" name="Account" duration-ms="3" started-at="2018-10-16T04:38:10Z" finished-at="2018-10-16T04:38:10Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping accountTest case. ]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping accountTest case. 
at CardholderMavenProject.Cardholder.Account(Cardholder.java:294)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Account -->
        <test-method status="PASS" signature="OrderCard1()[pri:3, instance:CardholderMavenProject.Cardholder@5bcab519]" name="OrderCard1" duration-ms="85641" started-at="2018-10-16T04:38:10Z" finished-at="2018-10-16T04:39:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderCard1 -->
        <test-method status="PASS" signature="LoadByBankTransfer()[pri:4, instance:CardholderMavenProject.Cardholder@5bcab519]" name="LoadByBankTransfer" duration-ms="58068" started-at="2018-10-16T04:39:36Z" finished-at="2018-10-16T04:40:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoadByBankTransfer -->
        <test-method status="SKIP" signature="Profile()[pri:5, instance:CardholderMavenProject.Cardholder@5bcab519]" name="Profile" duration-ms="1" started-at="2018-10-16T04:40:34Z" finished-at="2018-10-16T04:40:34Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping profileTest case. ]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping profileTest case. 
at CardholderMavenProject.Cardholder.Profile(Cardholder.java:386)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Profile -->
        <test-method status="PASS" signature="Statement()[pri:6, instance:CardholderMavenProject.Cardholder@5bcab519]" name="Statement" duration-ms="103280" started-at="2018-10-16T04:40:34Z" finished-at="2018-10-16T04:42:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Statement -->
        <test-method status="SKIP" signature="Suspend()[pri:7, instance:CardholderMavenProject.Cardholder@5bcab519]" name="Suspend" duration-ms="1" started-at="2018-10-16T04:42:17Z" finished-at="2018-10-16T04:42:17Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping suspendedTest case. ]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping suspendedTest case. 
at CardholderMavenProject.Cardholder.Suspend(Cardholder.java:541)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Suspend -->
        <test-method status="PASS" signature="Unload()[pri:8, instance:CardholderMavenProject.Cardholder@5bcab519]" name="Unload" duration-ms="86481" started-at="2018-10-16T04:42:17Z" finished-at="2018-10-16T04:43:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Unload -->
        <test-method status="PASS" signature="OrderCardNonEU()[pri:9, instance:CardholderMavenProject.Cardholder@5bcab519]" name="OrderCardNonEU" duration-ms="108778" started-at="2018-10-16T04:43:44Z" finished-at="2018-10-16T04:45:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderCardNonEU -->
        <test-method status="PASS" signature="LoadByCardEU()[pri:10, instance:CardholderMavenProject.Cardholder@5bcab519]" name="LoadByCardEU" duration-ms="124877" started-at="2018-10-16T04:45:32Z" finished-at="2018-10-16T04:47:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoadByCardEU -->
        <test-method status="FAIL" signature="LoadByCardNonEU()[pri:11, instance:CardholderMavenProject.Cardholder@5bcab519]" name="LoadByCardNonEU" duration-ms="83597" started-at="2018-10-16T04:47:37Z" finished-at="2018-10-16T04:49:01Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Expected condition failed: waiting for visibility of element located by By.cssSelector: div.alert-dismissible.alert-success.order-payment.alert.fade.in (tried for 30 second(s) with 500 milliseconds interval)]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of element located by By.cssSelector: div.alert-dismissible.alert-success.order-payment.alert.fade.in (tried for 30 second(s) with 500 milliseconds interval)
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:81)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:271)
at CardholderMavenProject.Cardholder.LoadByCardNonEU(Cardholder.java:1362)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate an element using By.cssSelector: div.alert-dismissible.alert-success.order-payment.alert.fade.in
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:32:14.902Z'
System info: host: 'DESKTOP-VUS56U8', ip: '192.168.2.100', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_181'
Driver info: driver.version: unknown
at org.openqa.selenium.support.ui.ExpectedConditions.lambda$findElement$0(ExpectedConditions.java:896)
at java.util.Optional.orElseThrow(Optional.java:290)
at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:895)
at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:44)
at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:206)
at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:202)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:248)
... 26 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoadByCardNonEU -->
        <test-method status="PASS" signature="OrderCardEU()[pri:12, instance:CardholderMavenProject.Cardholder@5bcab519]" name="OrderCardEU" duration-ms="86549" started-at="2018-10-16T04:49:01Z" finished-at="2018-10-16T04:50:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderCardEU -->
        <test-method status="SKIP" signature="PIN()[pri:13, instance:CardholderMavenProject.Cardholder@5bcab519]" name="PIN" duration-ms="5" started-at="2018-10-16T04:50:27Z" finished-at="2018-10-16T04:50:27Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping pinTest case. ]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping pinTest case. 
at CardholderMavenProject.Cardholder.PIN(Cardholder.java:629)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PIN -->
        <test-method status="PASS" signature="ViewImage()[pri:14, instance:CardholderMavenProject.Cardholder@5bcab519]" name="ViewImage" duration-ms="24320" started-at="2018-10-16T04:50:27Z" finished-at="2018-10-16T04:50:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ViewImage -->
        <test-method status="PASS" signature="OrderCardTest()[pri:15, instance:CardholderMavenProject.Cardholder@5bcab519]" name="OrderCardTest" duration-ms="52456" started-at="2018-10-16T04:50:52Z" finished-at="2018-10-16T04:51:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderCardTest -->
      </class> <!-- CardholderMavenProject.Cardholder -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
